space-framework/docs/space-framework.md

```diff
@@ -0,0 +1,92 @@
+   * **Unmanaged Space**: The SF supports discovery of existing pSpaces on a space provider and adding them as "Unmanaged Spaces" into the framework. This mode is similar to the import mode in terms of the "source of truth" - the desired state is defined by the pSpace. However, as oppose to import, here the pSpaces are automatically discovered through the space provider regular interface and the pSpace access information is retrieved from the  space provider similarly to how it is done in the managed space case. The SF supports pattern based filtering rules to limit the discovery of pSpaces. 
+  2. Providerless spaces: This spaces don't belong to a space provider and are called "Imported spaces". Such Space is created by importing an existing pSpace. In this case the Space is not linked to any space provider and the client which imports that Space need to supply the access information that allows other clients to connect to the pSpace. For imported spaces the desired state is derived by the pSpace. This means that the SF is not responsible for the life cycle management of the pSpace but only update the corresponding Space object. For example, when deleting the space object of an existing imported pSpace the space framework will recreate the space object. 
+### Space Manager 
+The Space Manager (SM) is a Kubernetes controller that is responsible for maintaining the state of the Space objects. The SM reconciles the Space and SpaceProviderDesc objects, and monitors the state of the pSpaces through the space provider.   
+The SM uses a library of space provider adaptors to communicate with the space providers. Currently the SF supports 3 space providers KCP, KIND, and KubeFlex, and includes 3 space provider adaptors that the SM uses.
+
+### Space provider adaptors
+The space manager uses a set of space provider adaptors that interacts with the space provider. All space provider adaptors implement a simple "space client interface" that includes basic pSpace life cycle operations. In addition the interface also defines the events that are sent from the adaptors to the space manager.
+
+### Space aware client (SAC)
+The Space aware client (SAC) allows clients/controllers to easily get access to the underlying space by simply using the space name. There is no direct interaction between the SAC and the Space  provider, and therefore the SAC is transparent to the specific provider of the spaces. 
```

Comment:
"simply using the space name" is not accurate, the `providerNS string` (which is actually specific to not just the provider but, more specifically, the SpaceProviderDesc --- right?

Reply:
Why was this marked as resolved?

Reply:
Because I changed to "by simply using the space name and optionally the SpaceProviderDesc name"

Reply:
But the second parameter is actually (a) the provider _namespace_ name and (b) not optional (as far as the reader of the interface sees).

Reply:
Will update the interface in the doc. To you request we added a another interface that uses the default provider so you don't need to specify it. We pass the provider name, there is a fixed rule that relates the providerNS and the provider name